lib/src/data/data-layer.ts

```diff
@@ -116,6 +118,19 @@ export class DataLayer<T> {
         return JSON.parse((await this._store.getData(this._resolveKey(key, userID))) ?? null);
     }
 
+    public setSessionStatus(status: string): void {
+        // Using local storage to store the session status as it is required to be available across tabs.
+        localStorage.setItem(`${ASGARDEO_SESSION_ACTIVE}`, status);
```

Comment:
@DonOmalVindula We will have to check whether `localStorage` available right? Especially in non-browser environments. WDYT?

Reply:
And  id the same funcationality is needed for other envs, may be we have to route towards an alternative way?

Reply:
@thivi

Reply:
Added a util function to check the storage availability before using in 3ca65be1d6fcce038c67546987aab3e5f12b646c